lib/index.js

```diff
@@ -13,6 +13,7 @@ var internals = {
     count: 0
 };
 
+exports.truncate = true;
```

Comment:
Instead of just `exports.truncate`, can you do `exports.settings` as an object. Then, in that object add a `truncateMessage` flag.